.vib/codeigniter/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - composer
+  - php-fpm
+  - gosu
+  - mmdblookup
+  - mysql
+files:
+  - paths:
+      - /opt/bitnami/codeigniter/app/Common.php
+      - /opt/bitnami/codeigniter/app/index.html
+      - /opt/bitnami/codeigniter/psalm-baseline.xml
+      - /opt/bitnami/codeigniter/psalm.xml
+      - /opt/bitnami/codeigniter/composer.json
+      - /opt/bitnami/codeigniter/spark
```

Comment:
I think we are checking for too many paths here. A simple minor version could rename or move some of these files to different locations, causing it to fail. Maybe it's worth to be a bit more granular?

Reply:
I removed spark because it's used in another tests anywy, and psalm-baseline.